packages/docs-site/src/content/docs/guides/indexers/subquery.md

```diff
@@ -0,0 +1,25 @@
+---
+title: SubQuery
+description: Indexing Taiko with SubQuery
+---
+
+SubQuery is a leading blockchain data indexer that provides developers with fast, flexible, universal, open source and decentralised APIs for web3 projects. SubQuery SDK allows developers to get rich indexed data and build intuitive and immersive decentralised applications in a faster and more efficient way. SubQuery supports many ecosystems including Taiko, Ethereum, Cosmos, Near, Polygon, Polkadot, Algorand, and more.
```

Comment:
Highlight specific features of SubQuery that are particularly beneficial for Taiko’s ZK-Rollup architecture, such as handling high throughput and low latency.